public/app/features/alerting/unified/NotificationPolicies.tsx

```diff
@@ -65,6 +65,7 @@ const AmRoutes = () => {
 
   const { selectedAlertmanager, hasConfigurationAPI, isGrafanaAlertmanager } = useAlertmanager();
   const { getRouteGroupsMap } = useRouteGroupsMatcher();
+  const { data: muteTimings } = useMuteTimings({ alertmanager: selectedAlertmanager ?? '' });
```

Comment:
maybe we should default to `GRAFANA_RULES_SOURCE_NAME` instead of an empty string?